setup.py

```diff
@@ -19,7 +19,7 @@
     "prettytable",
     # IPython dropped support for Python 3.8
     "ipython<=8.12.0; python_version <= '3.8'",
-    "ipython<8.16.0",
+    "ipython",
```

Comment:
remove this, we already have ipython as dependency (line 21), we don't need this one

Reply:
[Fixed](https://github.com/ploomber/jupysql/pull/878/commits/3cce632b8d4c8f162b3a09b87c6fade4db0d15e4)

Reply:
the link seems broken

![image](https://github.com/ploomber/jupysql/assets/989250/c11f2445-0d06-4e66-b602-bc8a11396245)

Reply:
[Fixed](https://github.com/ploomber/jupysql/pull/878/commits/d6002ac0e223e15ca368c27bf22b6bf70d821158)

Updated the link